circle.yml

```diff
@@ -1,23 +1,23 @@
+general:
+  artifacts:
+    - "tmp"
+
 machine:
+  environment:
+    ACCESSLINT_MASTER_BRANCH: dev
```

Comment:
Some more context on this setting: by default, accesslint looks at log artifacts in the most recent successful build on `master`, and compares those with the current PR build. That way, only new issues will get reported. Since 18F uses `master` as a production deploy branch and uses `dev` as a stable development branch, it makes more sense to point to the `dev` branch for comparisons. ACCESSLINT_MASTER_BRANCH provides this override.